apps/services/personal-representative-public/src/app/modules/personalRepresentatives/personalRepresentatives.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { AuthScope } from '@island.is/auth/scopes'
+import {
+  PersonalRepresentativeService,
+  PersonalRepresentativePublicDTO,
+} from '@island.is/auth-api-lib/personal-representative'
+import {
+  BadRequestException,
+  Controller,
+  UseGuards,
+  Get,
+  Inject,
+  Query,
+} from '@nestjs/common'
+import { ApiTags, ApiOperation, ApiBearerAuth } from '@nestjs/swagger'
+import { Documentation } from '@island.is/nest/swagger'
+import { Audit, AuditService } from '@island.is/nest/audit'
+import { environment } from '../../../environments'
+import {
+  Auth,
+  CurrentAuth,
+  IdsAuthGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+
+const namespace = `${environment.audit.defaultNamespace}/personal-representatives`
+
+@UseGuards(IdsAuthGuard, ScopesGuard)
+@Scopes(AuthScope.publicPersonalRepresentative)
+@ApiBearerAuth()
+@ApiTags('Personal Representatives - Public')
+@Controller('v1/personal-representatives')
+@Audit({ namespace })
+export class PersonalRepresentativesController {
+  constructor(
+    @Inject(PersonalRepresentativeService)
+    private readonly prService: PersonalRepresentativeService,
+    private readonly auditService: AuditService,
+  ) {}
+
+  /** Gets a personal representative rights by nationalId of personal representative */
+  @ApiOperation({
+    summary:
+      'Gets personal representative rights by nationalId of personal representative',
+    description: 'A personal representative can represent more than one person',
+  })
```

Comment:
You can use the @Documentation decorator below for this:
```typescript
@Documentation({
    summary:
      'Gets personal representative rights by nationalId of personal representative',
    description: 'A personal representative can represent more than one person',
})
```

Reply:
And for other places as well.
Don't need to specify ApiOperation

Reply:
`summary` just got merged to main ... so you need to rebase in order to access it.

Reply:
A ok was just about to comment that summary was not a part of the @documentation decorator :)